tests/PHPUnit/Fixtures/EmptySiteWithSiteContentDetectionGTM.php

```diff
@@ -31,7 +31,8 @@ public function provideContainerConfig()
             'ga4' => false,
             'gtm' => true,
             'cloudflare' => true,
-            'cms' => SitesManager::SITE_TYPE_WORDPRESS
+            'cms' => SitesManager::SITE_TYPE_WORDPRESS,
+            'react' => true
```

Comment:
Should this be `'jsFramework' => SitesManager::JS_FRAMEWORK_REACT` or possibly `'jsFramework' => SitesManager::JS_FRAMEWORK_UNKNOWN` if this test isn't meant to detect react?

Reply:
I need to update this, we added this to test that GTM is always shown as selected when multiple technologies are detected

Reply:
Removed this line from code